docs/features/models.md

```diff
@@ -29,14 +29,13 @@ on your model to disable this behavior.
 
 ### Automatic object ownership
 
-If you want to keep track of object ownership automatically, you can use the `CurrentUserMiddleware`.
-Just make sure, you'll insert it **after** djangos `AuthenticationMiddleware`:
+If you want to keep track of object ownership automatically, you can use the `CurrentRequestMiddleware`:
 
 ````python
 MIDDLEWARE = (
     ...
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
```

Comment:
The goal of this code example was initially to tell developers to put the CurrentUserMiddleware _after_ AuthenticationMiddleware - this specific order is no longer required because the user is now obtained on demand from the current request. In fact, a project might not use AuthenticationMiddleware at all, and instead solely rely on other mechanisms that provide `request.user` (as eg. DRF auth does), yet the middleware would still work.